packages/db-objects-redis/lib/objects/objectsInRedisClient.js

```diff
@@ -552,21 +538,53 @@ class ObjectsInRedisClient {
                 });
             }
 
-            this.log.debug(this.namespace + ' Objects client initialize lua scripts');
+            if (!this.client) {
+                return;
+            }
+
+            // for controller v4 we have to check if we can use the new lua scripts and set logic
+            // TODO: remove this backward shim if controller v4.0 is old enough
+            const keys = await this._getKeysViaScan(`${this.objNamespace}system.host.*`);
```

Comment:
sort out ones that have "deeper levels" then just host to not load all objects in system.host? :-)